rbx_types/src/attributes/mod.rs

```diff
@@ -0,0 +1,141 @@
+use crate::variant::VariantType;
+use std::convert::TryFrom;
+use thiserror::Error;
+
+mod reader;
+mod writer;
+
+pub use reader::get_attributes;
+pub use writer::attributes_from_map;
```

Comment:
```suggestion
use std::convert::TryFrom;

use thiserror::Error;

use crate::variant::VariantType;

mod reader;
mod writer;

pub use reader::get_attributes;
pub use writer::attributes_from_map;
```

We should prefer to keep our imports sorted into sections based on where they're from.